apriltag_ros/src/continuous_detector.cpp

```diff
@@ -86,6 +93,72 @@ bool ContinuousDetector::refreshParamsCallback(std_srvs::Empty::Request& req,
   return true;
 }
 
+void ContinuousDetector::publishEmptyDetection(const std_msgs::Header& header)
+{
+  AprilTagDetectionArray tag_detection_array;
+  tag_detection_array.header = header;
+  tag_detections_publisher_.publish(tag_detection_array);
+}
+
+bool ContinuousDetector::isTagInFOV(const tf::Transform& tag_pose) const
+{
+  if (!camera_model_.initialized())
+  {
+    ROS_WARN_STREAM_THROTTLE(1.0, "Camera not initialized, skipping FOV check");
+    return false;
+  }
+  if (tag_pose.getOrigin().getZ() <= 0)
```

Comment:
Add a line separation here:
```suggestion

  if (tag_pose.getOrigin().getZ() <= 0)
```